test/runtime/runtime-kotlin/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Gradle-specific files
+build/
+
+# Generated Src files
+src/test/kotlin/Address.kt
```

Comment:
Are you sure it's this file you want to ignore? 🤔 https://github.com/asyncapi/modelina/blob/0175034f1d5d3e422265b0ef8a799dd2bc4214a6/test/runtime/runtime-kotlin/src/test/kotlin/AddressTest.kt